benchmarks/solkz/solkz.cc

```diff
@@ -1,856 +1,4 @@
-#include <aspect/material_model/simple.h>
```

Comment:
Yeah, pretty much the refactoring was equivalent to the command `$ mv solkz.cc solkz.h` :smile: